WMass/python/plotter/mcPlots.py

```diff
@@ -747,7 +747,7 @@ def __init__(self,tdir,options):
         self._options = options
         self._dir = tdir
         ROOT.gROOT.ProcessLine(".x tdrstyle.cc")
-        ROOT.gROOT.ProcessLine(".L smearer.cc+")
+        # ROOT.gROOT.ProcessLine(".L smearer.cc+") # this should not be here, smearer has already been compiled (if smearer_cc.so not in ROOT.gSystem.GetLibraries())
```

Comment:
@cippy if this is the culprit of the problem, could you close this PR and open a new one with only this fix?